### ISSUE TITLE
Jump from launch to an initial intent

### ISSUE THREAD

**Issue by nikolasleblanc** [alexa-js/alexa-app#102]
If at the conclusion of an intent I would like to return to the initial intent, how would I do that?

```
app.launch(function(request,response) {
  // jump to InitIntent
});
app.intent('InitIntent', function(request, response) {
  // some intent stuff
});
```

**Comment by dblock** [comment]
Maybe ask this on an Alexa forum? (add link)

**Comment by nikolasleblanc** [comment]
It can definitely be done, just wondering if it was part of the alexa-app api, see https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs:

```
var handlers = {
    'LaunchRequest': function () {
        this.emit('HelloWorldIntent');
    },

    'HelloWorldIntent': function () {
        this.emit(':tell', 'Hello World!');
    }
 };
```

**Comment by dblock** [comment]
@nikolasleblanc Would welcome a new feature that does it!

**Comment by nikolasleblanc** [comment]
Cool @dblock. I'm _pretty_ new to this, but really enjoying it. Will see what I can do!

**Comment by ajcrites** [comment]
You can create a function that is used by both the initial intent and the launch intent. I think that it would currently be very difficult to write code that would trigger a handler (launch or otherwise) from another handler.

    app.launch((req, res) => {
      // do launch things
      return initialBehavior(req, res);
    });
    app.intent("HelloWorldIntent", (req, res) => {
      return initialBehavior(req, res);
    });
    function initialBehavior(req, res) {
      // do things you would do for both here
    }

**Comment by ajcrites** [comment]
I like the idea of being able to trigger other handlers from within a handler (launch as well as other intents). I'll try to hack on this when I get a chance to.

**Comment by kobim** [comment]
Resolved via #374
